moveit_simple/src/moveit_simple.cpp

```diff
@@ -895,7 +909,30 @@ bool Robot::toJointTrajectory(const std::string traj_name,
   return true;
 }
 
+bool Robot::computeIKTransforms()
+{
+  if (!virtual_robot_state_->knowsFrameTransform(ik_base_frame_))
```

Comment:
Throughout this class we use `tf` for getting frame transformations.  I believe this is the better approach because `tf` has access to more transforms than the `robot_state`.  I believe `robot_state` only knows about the frames defined in the URDF.

Reply:
Done.